<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>321</width>
     <height>221</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_5">
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>113</width>
       <height>184</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>111</width>
        <height>318</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_9">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QToolBox" name="toolBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>85</width>
       <height>143</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>85</width>
        <height>56</height>
       </rect>
      </property>
      <attribute name="label">
       <string>家人</string>
      </attribute>
      <widget class="QToolButton" name="toolButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>46</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <attribute name="label">
       <string>朋友</string>
      </attribute>
      <widget class="QToolButton" name="toolButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>46</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <attribute name="label">
       <string>黑名单</string>
      </attribute>
      <widget class="QToolButton" name="toolButton_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>46</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string> 百度</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>谷歌</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>     传智</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string> 抖音</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_scro">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>btn_scro</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tabbox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>btn_tabbox</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tabwidget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>btn_tabwidget</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tlj">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string> 选中拖拉机</string>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>320</y>
     <width>263</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>450</y>
     <width>104</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>104</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>450</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>490</y>
     <width>104</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>63</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>560</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDial" name="dial">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>540</y>
     <width>50</width>
     <height>64</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>360</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="verticalScrollBar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>16</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>450</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QKeySequenceEdit" name="keySequenceEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>530</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>170</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>68</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>0</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>290</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
